data/lists.js

```diff
@@ -3,3 +3,168 @@
 // edit list - used by /list/:id route
 // delete list - used by /list/:id route
 // delete entry from list - used by /list/:id route
+import { users } from "../config/mongoCollections.js";
+import { ObjectId } from "mongodb";
+import * as validation from "../utils/validation.js";
+
+const exportedMethods = {
+  async create(userId, title, movies) {
+    //for validation validate if
+    //userId is valid
+    //title is a string
+    //movies is an array of integers
+    validation.checks(userId);
+    userId = userId.trim();
+    if (!ObjectId.isValid(userId)) throw "invalid object ID";
```

Comment:
Lines 16-18, there is already a validation helper to check for IDs. You can replace those three lines with
```suggestion
    userId = validation.checkId(userId, "User ID")
```